### ISSUE TITLE
Integration tests for miners

### ISSUE THREAD

**Issue by hide-on-bush-x** [masa-finance/masa-bittensor#119]
### Based on #99 
It is required that we have some scripts to validate that the miner works as expected

Testing list
- Blacklist ( check that the miner doesn't answer calls from axons that are not part of the network )
- Blacklist ( check that the miner doesn't answer calls from axons that dont follow the blacklist criteria ( staking minimums etc ) )
- Response accuracy  ( Call the masa protocol with a specific prompt, then call the miner with that prompt and compare results )

More info about this on #99

**Comment by juanmanso** [comment]
Follow up on #118, is this:
- Blocked by #117 ?

**Comment by Luka-Loncar** [comment]
We need to groom this on the next planning and get final agreement between eng team and devops on how the integration tests should look like and how they should be executed.

**Comment by Luka-Loncar** [comment]
Closing this because it is covered in #120 and #121
